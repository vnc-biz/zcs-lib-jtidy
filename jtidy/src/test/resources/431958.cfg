# Tidy configuration file for bug #431958
indent: auto
tidy-mark: false

indent= auto
tidy-mark= false