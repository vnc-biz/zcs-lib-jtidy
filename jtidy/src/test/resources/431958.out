<html>
<head>
<title>[ #431958 ] Comments always indented</title>
</head>
<body>
   <!--
   -->
</body>
</html>
